examples/esp32spi/minimqtt_pub_sub_blocking_topic_callbacks_esp32spi.py

```diff
@@ -94,8 +93,8 @@ def on_message(client, topic, message):
 
 # Set up a MiniMQTT Client
 client = MQTT.MQTT(
-    broker=os.getenv("broker"),
-    port=os.getenv("broker_port"),
+    broker=broker,
+    port=broker_port,
     socket_pool=pool,
```

Comment:
Think we want to add the username and password here too, not sure how this ever worked unless it was a broker that doesn't require one (like test.mosquitto.org), but the aio_username used in topic later is now needed so we might as well offer it here too.
```suggestion
    broker=broker,
    port=broker_port,
    username=aio_username,
    password=aio_key,
    socket_pool=pool,
```